CMakeLists.txt

```diff
@@ -506,6 +511,13 @@ macro(nebula_link_libraries target)
     )
 endmacro(nebula_link_libraries)
 
+function(nebula_add_subdirectory dir_name)
+    if ((NOT ENABLE_TESTING) AND (${dir_name} STREQUAL test))
```

Comment:
I think it's better not to ignore the entire `test` directory, but rather to provide a macro like `nebula_add_test` to ignore a single UT.

Reply:
In addition to binary, there are also objects. We specify the `test` directory as the directory of tests

Reply:
Do all tests have to be placed in the test directory? How to disable one or more of these tests ?

Reply:
@yixinglu To assume that all unit tests are placed in _**test**_ directory is not a strong assumption.

For the time being, we have not demands to disable the building of individual tests.

Reply:
Got it!